sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v3/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+---
+friendly_name: Sanitized Firefox Search Impression Data
+owners:
+  - ctroy@mozilla.com
+labels:
+  incremental: true
+description: |-
+  Firefox Suggest impression pings
+  passed through the keyword list, then
```

Comment:
```suggestion

```

I believe we can remove the bit about it being passed through the keyword list

Reply:
Can do!